core/templates/components/state-directives/answer-group-editor/summary-list-header.component.ts

```diff
@@ -12,13 +12,18 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import { Component, Input } from '@angular/core';
+import { Component, EventEmitter, Input, Output } from '@angular/core';
 import { downgradeComponent } from '@angular/upgrade/static';
 
 /**
  * @fileoverview Component for the header of items in a list.
  */
 
+interface EventData {
```

Comment:
EventData seems a bit too generic. Maybe we could rename it to provide more information on what event would this data be attached to. Something like `deleteSummaryEventData`?

Reply:
I am changing it to `DeleteSummaryEventData` taking into account that interface name should start with capital Letter.